src/solver/batchnorm/backward_spatial_single.cpp

```diff
@@ -147,14 +148,15 @@ BnBwdTrainingSpatialSingle::GetSolution(const ExecutionContext& context,
                 variant    = 3;
                 xlocalsize = 1024;
                 xgridsize  = c * xlocalsize;
-                ldsgcn     = xlocalsize / 64;
+                ldsgcn     = xlocalsize / wavesize;
                 ldsnogcn   = xlocalsize;
             }
             else
             {
                 variant    = 0;
                 xlocalsize = 1024;
-                ldsgcn     = xlocalsize / 64;
+                xgridsize  = 1024 * c;
```

Comment:
:red_circle: Aha, I see, this is the fix. xgridsize wasn't initialized in previous commit!